js/aboutApp/renderer.js

```diff
@@ -19,10 +19,12 @@ function renderDom() {
 
 ipcRenderer.on('versionInfo', (event, versionInfo) => {
     const versionText = document.getElementById('version');
-    const { version, clientVersion, buildNumber } = versionInfo;
+    const { version, clientVersion, buildNumber, i18n } = versionInfo;
```

Comment:
Can be destructed as below with default values to prevent undefined values.
```suggestion
    const { version, clientVersion, buildNumber, 'About Symphony': aboutSymphony = 'About Symphony', 'Version': ver = 'version' } = versionInfo;
```